tools/mwasmarm_patcher/mwasmarm_patcher.c

```diff
@@ -0,0 +1,70 @@
+#include <stdint.h>
+#include <stdio.h>
+#include <openssl/sha.h>

```

Comment:
We shouldn't require openssl just for a hash function.

Reply:
Give me an sha1 implementation thats freestanding and I will update it. Otherwise it's going in.